### PR TITLE
fixes #2426 - translate model and column names to English

### PR DESCRIPTION
**Pull request by domcleal**
- "translated" all of the English model and column names
- one big change was renaming "Hostgroup" to "host group"
- tried to remove extraneous capital letters from sentences around the app (incomplete, but better)
- improved taxonomies i18n by extracting variations of the taxonomy/controller name

### CODE REVIEW

File: app/helpers/taxonomy_helper.rb [hunk 1]
```diff
@@ -29,16 +29,16 @@ def location_dropdown count
       end
   end
 
-  def taxonomies_plural
-    controller_name
```
Comment (ohadlevy) [inline app/helpers/taxonomy_helper.rb:33]: why remove this? was this unused?

File: app/views/taxonomies/index.html.erb [hunk 2]
```diff
@@ -20,14 +20,14 @@
   <% @taxonomies.each do |taxonomy| %>
     <tr class="<%= cycle("even", "odd") -%>">
       <td><%= link_to taxonomy.to_label, edit_taxonomy_path(taxonomy) %></td>
-      <td><%= link_to @counter[taxonomy.id] || 0, hosts_path(:search => "#{taxonomy_single} = #{taxonomy}") %></td>
+      <td><%= link_to @counter[taxonomy.id] || 0, hosts_path(:search => "#{controller_name.singularize} = #{taxonomy}") %></td>
```
Comment (ohadlevy) [inline app/views/taxonomies/index.html.erb:23]: are we planning to translate the search / autocompleter?

**Comment by ohadlevy** [comment]
merged as 9a28016 with minor whitespaces changes

File: app/helpers/taxonomy_helper.rb (hunk 1 above)
Comment (domcleal) [inline app/helpers/taxonomy_helper.rb:33]: yes, unused (or perhaps it was used but should have been single, I can't remember)

File: app/views/taxonomies/index.html.erb (hunk 2 above)
Comment (domcleal) [inline app/views/taxonomies/index.html.erb:23]: It's on the Translating wiki page list, so possibly.  For now I've just been trying to make sure we always use the english terms in any search queries embedded in the app so they still work.

**Comment by lzap** [comment]
Nicely done +1

**Pull request closed without merging**